API-1423: Add warning (#461)
akeneo/pim-api-docs: content/guides/events-api-connection/warning.md

> # Limitations

Reviewer: We should probably only add a link to the existing docs https://api.akeneo.com/events-documentation/limits-and-scalability.html
And to be honest I don't understand why we don't add theses limits to the existing docs

## No retry
In some message brokers, the retry mechanism is natively implemented. 
What does it mean? In brief, when a message publication failed, the system will automatically reattempt it later. 

> Very useful, isn’t it? Especially in case of temporarily network issues: the message won’t be lost.

Reviewer 2: I would remove the l.4, 5 and 6.

Unfortunately, we don’t have such a retry mechanism in our event API connection. 
This means that if a message is not published, no mechanism will try to do it again, and the message will be forever lost...

## Requests limitation

> For some performance issues, the number of requests is limited to 4000 per hour.

Reviewer: We should probably only add a link to the existing docs https://api.akeneo.com/events-documentation/limits-and-scalability.html

> For some performance issues, the number of requests is limited to 4000 per hour.

Reviewer 2: ```suggestion
For some scalability issues and to avoid overflowing the endpoint, the number of requests is limited to 4000 per hour.
```

## No 100% uptime

> Since event API delivery is not always guaranteed, don't blindly trust it and consider implementing a job which periodically fetch data from the PIM.

Reviewer 3: ```suggestion
Since event API delivery is not always guaranteed, you should consider implementing a job which periodically fetch data from the PIM.
```

> Since event API delivery is not always guaranteed, don't blindly trust it and consider implementing a job which periodically fetch data from the PIM.

Reviewer 2: ```suggestion
Since event API delivery is not always guaranteed, don't blindly trust it and consider implementing a job that periodically fetches data from the PIM in order to ensure the synchronization of catalogs.
```
